lib/src/lightclient.rs

```diff
@@ -25,7 +25,7 @@ use crate::grpcconnector::{self, *};
 use crate::SaplingParams;
 use crate::ANCHOR_OFFSET;
 
-mod checkpoints;
+use crate::checkpoints;
```

Comment:
I didn't understand this. Why should we move the checkpoints out of the lightclient?